macOS/DuckDuckGoVPN/TunnelControllerIPCService.swift

```diff
@@ -192,24 +202,16 @@ extension TunnelControllerIPCService: XPCServerInterface {
 
     func start(completion: @escaping (Error?) -> Void) {
         Task {
+            defer { completion(nil) }
```

Comment:
This is so that the IPC response to the tunnel start attempt waits until the start attempt really concludes.